packages/contracts/src/resources/site.ts

```diff
@@ -158,11 +168,32 @@ export const siteApi = contract.router(
 				},
 			},
 		},
+		searchUsers: {
+			path: "/searchUsers",
+			method: "GET",
+			summary: "Get a list of matching users for autocomplete",
+			description:
+				"Get a list of users matching the provided query. Used for rendering suggestions in an autocomplete input for selecting users.",
+			query: z.object({
+				communityId: communitiesIdSchema,
+				email: z.string(),
+				name: z.string().optional(),
+				limit: z.number().optional(),
+			}),
+			responses: {
+				200: safeUserSchema
+					.extend({ member: communityMembershipsSchema.nullable().optional() })
+					.array(),
+			},
+		},
```

Comment:
very sorry, i overlooked this in my initial review, but could we make this endpoint slightly more in line with our other endpoints?

main things i'm thinking of

- instead of `searchUsers`, make it `users.search` (?).
- similarly, change the `path` to match, eg `/users/search`. 
- don't require the `communityId` in the query object. since the api routes are scoped to a community we can derive the communityid that way (see my comment in the route)

related to the above: do you think it makes sense to have separate `/users` and `/members` endpoints? I do think well add a way to get all (community) members in a community, but should that be `GET /members` or `GET /users`? We can decide that later i think, but interested in your thoughts

Reply:
i'm happy to do whatever you think is best with the api! i named it this way because i figured it would behave a little differently than the other get endpoints, but i think users.search vs users.get captures that well enough. and i'll make the other changes!

Reply:
i'm not really sure about members vs users. i want to modify the member schema type to store a userid in the future since not everyone will have a community membership (though I would like to change that too). but other than that i don't really know what else we'll be using these endpoints for yet